### ISSUE TITLE
QUESTIONS - Dataset, linear classifier, plotting

### ISSUE THREAD

**Issue by timothyvanco** [ReshinthAdith/BYOL-Pytorch#1]
Hi, I looked on your code to BYOL and it looks so good. I am a bit new to this problematic of Self-supervised learning, but I have a few questions:
1.) Is it possible to use this code with my own dataset? For example, I would have dataset of dogs and cats etc.. and I want to use it instead of CIFAR10 as you used.
2.) Why you did not try to test your trained model? Is it possible to be done with building a Linear classifier on top of a model and then finetune it?
3.) Are you planning to upgrade your code to plot accuracy or loss?
4.) Is your loss function written well? Because you have:
`Loss = -2 * (Norm_Rep1*Norm_Rep2).sum(dim=-1)`
but sthalles (which is your source code you wrote) have it like this:
`loss = 2 - 2 * (prediction_norm * target_norm).sum(dim=-1)`

Thanks and have a nice day :)
